scenariolib/config.go

```diff
@@ -89,13 +89,13 @@ type Config struct {
 
 	// DefaultOriginLevel3 Override of the default OriginLevel3.
 	DefaultOriginLevel3 string `json:"defaultOriginLevel3,omitempty"`
-}
 
-// RandomData An override of the bot default random/fake data.
-type RandomData struct {
 	// DefaultPageViewField Override of the DefaultPageViewField for ALL pageView Events.
 	DefaultPageViewField string `json:"defaultPageViewField,omitempty"`
```

Comment:
This one makes sense since we have other defaults in the top level of the config.